config/settings.yml

```diff
@@ -79,4 +79,7 @@ scanned_exams:
 python:
   bin: <%= ::Rails.root %>/venv/bin
 
+exception_notification:
+  enabled: false
```

Comment:
Provide default values for the other settings as well, to illustrate them (you can use `markus-exception-notification@example.com` and `markus-admin@example.com` for the sender and recipient, respectively).